sdk/containerregistry/arm-containerregistry/test/containerregistry_examples.ts

```diff
@@ -159,94 +159,94 @@ describe("ContainerRegistry test", () => {
   //   assert.equal(resArray.length,0);
   // });
 
-  it("tasks create test", async function () {
-    const res = await client.tasks.beginCreateAndWait(resourceGroup, registryName, taskName, {
-      location: location,
-      tags: {
-        testkey: "value"
-      },
-      status: "Enabled",
-      platform: {
-        os: "Linux",
-        architecture: "amd64"
-      },
-      agentConfiguration: {
-        cpu: 2
-      },
-      step: {
-        type: "Docker",
-        contextPath: "https://github.com/SteveLasker/node-helloworld",
-        imageNames: ["testtask:v1"],
-        dockerFilePath: "DockerFile",
-        isPushEnabled: true,
-        noCache: false
-      },
-      trigger: {
-        baseImageTrigger: {
-          name: "myBaseImageTrigger",
-          baseImageTriggerType: "Runtime",
-          updateTriggerPayloadType: "Default",
-          status: "Enabled"
-        }
-      }
-    }, testPollingOptions);
-    assert.equal(res.name, taskName);
-  });
+  // it("tasks create test", async function () {
```

Comment:
this change is because of https://github.com/Azure/azure-sdk-for-js/pull/25001/files#diff-d1f3e61042d73e2ed235f777f4ac5ceef091e923c483bf7b6fc1e5cd24d6c01aR92